src/main/java/uk/gov/justice/digital/job/batchprocessing/BatchProcessor.java

```diff
@@ -31,95 +30,49 @@ public class BatchProcessor {
 
     private static final Logger logger = LoggerFactory.getLogger(BatchProcessor.class);
 
-    private final RawZone rawZone;
     private final StructuredZoneLoad structuredZoneLoad;
-    private final StructuredZoneCDC structuredZoneCDC;
     private final CuratedZoneLoad curatedZoneLoad;
-    private final CuratedZoneCDC curatedZoneCDC;
-    private final SourceReferenceService sourceReferenceService;
-    private final ViolationService violationService;
+    private final ValidationService validationService;
 
     @Inject
     public BatchProcessor(
```

Comment:
This is actually a delete of the old BatchProcessor and a rename of S3BatchProcessor to BatchProcessor so the diff is a bit misleading.